src/core/include/openvino/pass/make_sdpa.hpp

```diff
@@ -0,0 +1,22 @@
+// Copyright (C) 2018-2024 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include "openvino/pass/graph_rewrite.hpp"
+#include "transformations_visibility.hpp"
+
+namespace ov {
+namespace pass {
+
+class TRANSFORMATIONS_API MakeSDPA;
```

Comment:
is it required to be in public headers?

Reply:
@ilya-lavrenov yes, we added them to public headers for linking with tests which we added recently

Reply:
public API is required for users, not for tests